sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/V2ScanRelationPushDown.scala

```diff
@@ -308,7 +317,7 @@ object V2ScanRelationPushDown extends Rule[LogicalPlan] with PredicateHelper {
     case other => other
   }
 
-  def applyLimit(plan: LogicalPlan): LogicalPlan = plan.transform {
+  def pushDownLimit(plan: LogicalPlan): LogicalPlan = plan.transform {
```

Comment:
the name `pushDownLimit` is same as `pushDownLimit` above. Can we use a different name or at least add some docs?

Reply:
Hm, maybe `pushdownLimits` is better.